src/components/AddTodo/AddTodo.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import { addTask, deleteTasks } from 'service/taskService';
+import { isValidValueInput } from 'helpers/validation';
+import './style.scss';
+
+const AddTodo = ({ allTasks, setAllTasks }) => {
+  const [userInput, setUserInput] = useState('');
+
+  const handleKey = (e) => {
+    if (e.key === "Enter") {
+      addNewTask(userInput);
+    };
+  };
+
+  const addNewTask = async() => {
+    try {
+      if (!isValidValueInput(userInput)) {
+        setUserInput('');
```

Comment:
Whether your validation checks if the field is empty. Why then clear it if it is already empty